app/controllers/likes_controller.rb

```diff
@@ -0,0 +1,19 @@
+class LikesController < ApplicationController
+  def create
+    puts 'I am in the create method'
+    @user = current_user
```

Comment:
- [ ] **[Optional]** I suggest you remove what you have on line `3` which was used for debugging purposes.